Project.toml

```
@@ -35,14 +35,13 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 AbstractMCMC = "0.5.2"
 AdvancedHMC = "0.2.20"
 AdvancedMH = "0.4"
-Bijectors = "0.6.2"
+Bijectors = "~0.6.2"
 BinaryProvider = "0.5.6"
 Distributions = "0.22"
-DistributionsAD = "0.4.3"
+DistributionsAD = "~0.4.3"
```

Same here, the compatible versions are the same as the ones you get with the previous
```suggestion
DistributionsAD = "0.4.3"
```

---

Well I learned something new today...